src/pkcs15init/pkcs15-lib.c

```diff
@@ -2683,7 +2697,8 @@ prkey_fixup_rsa(struct sc_pkcs15_card *p15card, struct sc_pkcs15_prkey_rsa *key)
 				key->iqmp.len = 0;
 			}
 		}
-
+end:
+		sc_log_openssl(ctx);
```

Comment:
I think this log should not be here as all the `goto end` have logged the errors already and this code path is executed also in case of success.